src/hotspot/share/opto/loopUnswitch.cpp

```diff
@@ -423,40 +530,25 @@ void PhaseIdealLoop::trace_loop_unswitching_impossible(const LoopNode* original_
   }
 }
 
-void PhaseIdealLoop::trace_loop_unswitching_count(IdealLoopTree* loop, LoopNode* original_head,
-                                                  const Node_List& unswitch_iffs) {
+void PhaseIdealLoop::trace_loop_unswitching_count(IdealLoopTree* loop, LoopNode* original_head) {
   if (TraceLoopOpts) {
     tty->print("Unswitch   %d ", original_head->unswitch_count() + 1);
     loop->dump_head();
-    for (uint i = 0; i < unswitch_iffs.size(); i++) {
-      unswitch_iffs.at(i)->dump(3);
-      tty->cr();
-    }
```

Comment:
Removed to reduce noise in favor of `TraceLoopUnswitching` (see `PhaseIdealLoop::trace_loop_unswitching_result()`).